docs/src/knownissues.md

```diff
@@ -34,6 +34,12 @@ This can be worked around be either:
 
 2. Launching julia with the  `--compiled-modules=no` option. This can result in much longer package load times.
 
+## Open MPI
+
+### Segmentation fault
+
+When attempting to use a system-provided Open MPI implementation, you may encounter a segmentation fault. This can be fixed by setting the environment variable `ZES_ENABLE_SYSMAN=1`. See [Open MPI issue #10142](https://github.com/open-mpi/ompi/issues/10142).
```

Comment:
This doesn't happen with `OpenMPI_jll`?

Reply:
Yes because `OpenMPI_jll` uses an embedded HWloc instead of `HWloc_jll`

```
julia> dlopen("/home/vchuravy/.julia/artifacts/aae9ba66be9f6bccbfa324bbab4fa317b676b9d2/lib/libhwloc.so")
Ptr{Nothing} @0x0000000001ec63a0

julia> dlclose(ans)

````

has this exact problem.

Reply:
From https://github.com/open-mpi/ompi/issues/10142#issuecomment-1072367339:

> Open MPI embeds a copy of hwloc (which, for the purposes of this discussion, is just a library), but that embedded copy of hwloc is only used if hwloc is not already available on your system. The version of hwloc that is included in Open MPI v4.1.2 is hwloc v2.0.2, which is long before the ZES_ENABLE_SYSMAN issue (i.e., it won't crash because of this).

Reply:
note that even if you are using a recent `hwloc` release that has this issue, the crash only occurs when `hwloc` is dlclose'd **and** the environment is accessed.

Reply:
hwloc 2.7.1rc1 is available from https://www.open-mpi.org/software/hwloc/v2.7/
I'll release the final 2.7.1 in a couple days in you can confirm the issue is gone (it should only exist with hwloc >=2.5 and <=2.7.0).